src/GraphQL/Resolver.hs

```diff
@@ -193,11 +193,13 @@ instance forall m. (Functor m) => HasGraph m Text where
   buildResolver handler _ =  map (ok . toValue) handler
 
 
-instance forall m hg. (Applicative m, HasGraph m hg) => HasGraph m (API.List hg) where
-  type Handler m (API.List hg) = [Handler m hg]
-  buildResolver handler selectionSet = map aggregateResults a
-    where
-      a = traverse (flip (buildResolver @m @hg) selectionSet) handler
+instance forall m hg. (Monad m, Applicative m, HasGraph m hg) => HasGraph m (API.List hg) where
+  type Handler m (API.List hg) = m [Handler m hg]
```

Comment:
We talked in person. We can't eliminate the inner one, and we need the outer one for, e.g., database queries.